README.md

```diff
@@ -75,7 +75,7 @@ If you are using Maven, add this to your pom.xml file
       <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>libraries-bom</artifactId>
-        <version>2.7.1</version>
+        <version>2.9.0</version>
```

Comment:
can you please add same changes in other clients as well.

Reply:
There's another PR for that.